connector/connect/common/pom.xml

```diff
@@ -140,6 +140,35 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Why do we need to shade common? For the off-chance someone takes a direct dependency on it?